megazords/full/Cargo.toml

```diff
@@ -21,5 +21,6 @@ nimbus-sdk = { path = "../../components/nimbus" }
 autofill = { path = "../../components/autofill" }
 crashtest = { path = "../../components/crashtest" }
 error-support = { path = "../../components/support/error" }
+merino_client = { path = "../../components/merino_client" }
```

Comment:
"Megazording" means "bundling all our Rust components in a single library" 🤖 Here's a [quick overview](https://mozilla.github.io/application-services/book/design/megazords.html) about it, and why we do it!